lib/libyear_bundler/models/gem.rb

```diff
@@ -35,9 +37,17 @@ def release_date(gem_name, gem_version)
             report_problem(gem_name, "Could not find release date for: #{gem_name}")
             return nil
           end
-          tup, source = tuples.first # Gem::NameTuple
-          spec = source.fetch_spec(tup) # raises Gem::RemoteFetcher::FetchError
-          spec.date.to_date
+          tup, = tuples.first # Gem::NameTuple
+          uri = URI.parse(
+            "https://rubygems.org/api/v2/rubygems/#{gem_name}/versions/#{tup.version}.json"
+          )
+          response = if http
```

Comment:
But `http` is always present here? Is this check needed? And is `http = nil` in all the declarations needed?

Reply:
Can do even better - just assign it to a constant and there won't be a need to pass it to all the methods.